### DISCUSSION TITLE
SQLite資料庫學習網站

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#501] [General]
因期末專題有用到SQLite，這是我自己找的學習資源，在Python使用SQLite相當簡單，Python有內建sqlite3套件可以使用，網站裡面基本上會用到的語法都有！
題外話：如果覺得用Python程式碼print出資料庫的樣子很麻煩，可以使用DB Browser for SQLite來管理資料庫，圖形化介面相當方便，可以節省不少為了熟悉SQL語法而浪費的時間，Windows、macOS、Linux都可以使用，而且還是免費的，心動不如馬上行動，資料庫的應用可以讓資料更有系統化的被儲存，搜尋也會更快！

https://www.runoob.com/sqlite/sqlite-tutorial.html

**Comment by NcuJerryKao** [comment]
[圖形化介面相當方便，可以節省不少為了熟悉SQL語法而浪費的時間]
這句真的太扯了 這是我初學語言熟悉語法 都會遇到的問題 有這功能太好了

**Comment by Ben6uo** [comment]
感謝大哥的分享 這對我的期未報告太好用了

**Comment by 32844583** [comment]
不過 SQLite 有很多缺點，SQLite 只能被本地程式訪問。如果需要在網路上訪問資料庫，則需要使用其他解決方案(MySQL 或 PostgreSQL)。

**Comment by manipulateed** [comment]
雖然學過mysql 但還是推推這學期老師教到的比較輕量級的資料庫～這篇文章更給出了如何去結合Python 的用法，非常有幫助

**Comment by gnsJhenJie** [comment]
其實SQLite跟MySQL和MariaDB的語法絕大部分都是一樣的，所以在Google相關語法時，可以直接搜尋MySQL的語法，可以找到的資訊會比較多。

**Comment by BingRong1217** [comment]
感覺相當實用也能節省很多時間

**Comment by limitx0** [comment]
推推，這網站寫的教學很好

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by larry40305** [comment]
對我個人而言，資料庫不僅僅是一個重大的事件，還可能會改變我的人生。
